lib/recipient.rb

```diff
@@ -0,0 +1,58 @@
+require 'dotenv'
+require 'json'
+
+# require_relative '../lib/bot_settings.json'
+Dotenv.load
+
+class SlackApiError < StandardError; end
+class NoMessageError < StandardError; end
+
+class Recipient
+  attr_reader :id, :name
+
+  def initialize(id, name)
+    @id = id
+    @name = name
+  end
+
+  API_KEY = ENV["SLACK_API_TOKEN"]
```

Comment:
The README said that this should be named `SLACK_TOKEN`:

```suggestion
  API_KEY = ENV["SLACK_TOKEN"]
```